src/components/study-pane.js

```diff
@@ -155,20 +145,20 @@ const StudyPane = ({
         }
     }, [network, filteredNominalVoltages, dispatch]);
 
-    function openVoltageLevelDiagram(vlId, substationId) {
+    function openVoltageLevelDiagram(vlId) {
         // TODO code factorization for displaying a VL via a hook
         if (vlId) {
             props.onChangeTab(0); // switch to map view
-            showVoltageLevelDiagram(vlId); // show voltage level
+            openDiagramView(vlId, SvgType.VOLTAGE_LEVEL);
         }
     }
 
     const openVoltageLevel = useCallback(
```

Comment:
openVoltageLevelDiagramFromMap ?
Those names are a bit long but that could be benefic to make a difference between the 2 methods

Reply:
This is not my code, I don't think I should modify it in this refactoring. Maybe in another US/TS